recipes/cppinclude/recipe.yaml

```
@@ -0,0 +1,51 @@
+context:
+  name: cppinclude
+  version: "0.5.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/cppinclude/cppinclude/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 31c36a347b43ee667c6f17121c2d0fb7cf4d2672130f25273a48835b21f6ae40
+  patches:
+    - 01_std_hash.diff
+
+build:
+  number: 0
+  skip:
+    - win
+    - osx
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - cmake
+    - make
+    - fmt 7.*
+  run:
+    - fmt 7.*
```

We pin packages globally. Can you remove this pin as suggested here
```suggestion
    - fmt
```